### PR TITLE
s5cmd: add version 2.0.0

### PR DESCRIPTION
**Pull request by cr1cr1**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

image

### CODE REVIEW

File: bucket/s5cmd.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            "hash": "92f2727b78ff3d0d982da38506dd475ba72b6f36ffcf607bba6185b0e9398981"
+        },
+        "64bit": {
+            "url": "https://github.com/peak/s5cmd/releases/download/v2.0.0/s5cmd_2.0.0_Windows-64bit.zip",
+            "hash": "c6505331f0e8d9337a3bf376df74fa7f23082ff9d57714d85171d079e25472da"
+        }
+    },
+    "bin": [
+        "s5cmd.exe"
+    ],
```
Comment (rashil2000) [inline bucket/s5cmd.json:18]:
```suggestion
    "bin": "s5cmd.exe",
```

**Comment by rashil2000** [comment]
Also add the Arm64 variant

File: bucket/s5cmd.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        },
+        "arm64": {
+            "url": "https://github.com/peak/s5cmd/releases/download/v2.0.0/s5cmd_2.0.0_Windows-arm64.zip",
+            "hash": "5ce05a2085d18b8debfb3788c633290435e39aa754c3bf37c417e6736ded2af6"
+        }
+    },
+    "bin": "s5cmd.exe",
+    "checkver": {
+        "url": "https://github.com/peak/s5cmd/tags",
+        "regex": "/releases/tag/v([\\d.]+)"
```
Comment (rashil2000) [inline bucket/s5cmd.json:23]:
```suggestion
        "regex": "/releases/tag/v([\\d.]+)\""
```

Comment (cristichiru) [reply bucket/s5cmd.json]: Perhaps the bot is not updated to take into account `checkver: github` but... whatever works

**Pull request merged**